src/Components/Server/src/Circuits/RemoteRenderer.cs

```diff
@@ -349,15 +358,15 @@ static Log()
                     EventIds.SendBatchDataFailed,
                     "Sending data for batch failed: {Message}");
 
-                _completingBatchWithError = LoggerMessage.Define<long, string>(
+                _completingBatchWithError = LoggerMessage.Define<long, string, double>(
                     LogLevel.Debug,
                     EventIds.CompletingBatchWithError,
-                    "Completing batch {BatchId} with error: {ErrorMessage}");
+                    "Completing batch {BatchId} with error: {ErrorMessage} in {ElapsedTime}ms.");
 
-                _completingBatchWithoutError = LoggerMessage.Define<long>(
+                _completingBatchWithoutError = LoggerMessage.Define<long, double>(
                     LogLevel.Debug,
                     EventIds.CompletingBatchWithoutError,
-                    "Completing batch {BatchId} without error");
+                    "Completing batch {BatchId} without error in {ElapsedTime}ms.");
```

Comment:
What do you think? Is this interesting to track:

```
dbug: Microsoft.AspNetCore.Components.Web.Rendering.RemoteRenderer[105]
      Completing batch 2 without error in 579.6303ms.
```

Reply:
This kind of thing is always nice to have. That's a lot of milliseconds though.

Reply:
I was likely debugging the app? It's usually not this bad 😄